resources/assets/uad_lists.json

```
@@ -37398,5 +37398,13 @@
     "neededBy": [],
     "labels": [],
     "removal": "Recommended"
+  },
+  "com.preff.kb.xm":{
+    "list": "Oem",
+    "description": "'Emoji & Font Keyboard' on Xiaomi. \nXiaomi won't let me uninstall it. If it's same for you, consider disabling it instead.",
```

```suggestion
    "description": "'Emoji & Font Keyboard' on Xiaomi.\nXiaomi won't let me uninstall it. If it's same for you, consider disabling it instead.",
```
If you disable it, then you'll be left without a keyboard at all right? Maybe suggest installing an alternative first. Is it like any other keyboard or just for emojis and fonts? Because if it's a general keyboard, it should be `Advanced`, see https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/wiki/FAQ.

---

done. 
also there shouldnt be any issue if the tool fails to uninstall it right? adb tool gives "FALIURE [-1000] when trying to uninstall it."

---

You can't uninstall all the apps. I assume disabling works? See https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/wiki/FAQ#how-is-it-possible-to-delete-a-system-package-without-root-permission.